tdrs-backend/tdpservice/parsers/parse.py

```diff
@@ -75,21 +85,48 @@ def parse_datafile(datafile):
             continue
 
         schema = get_schema(line, section, schema_options)
-        record_is_valid, record_errors = parse_datafile_line(line, schema)
 
-        if not record_is_valid:
-            errors[line_number] = record_errors
+        if isinstance(schema, util.MultiRecordRowSchema):
+            records = parse_multi_record_line(line, schema)
+
+            n = 0
```

Comment:
Since this is for multi-record lines, n doesn't correlate with line number but rather interline record count, correct? Could we alter the name for clarity?

Alternatively, do we even need a named entry? What's the advantage/is this used? I believe we keep multiple errors for ordinary lines, why not just keep that same structure and get the info needed from ParserError which has name, startIndex, case number?

Reply:
the goal was just to add some structure/clarity to the returned error object for my own testing. you're right in that we don't really use the returned object, and flattening it would still contain all the relevant data. if you'd like to discuss the structure of the returned errors object, i'm open to having a broader conversation about the entire thing.